srcs/app/server/Server.cpp

```diff
@@ -16,6 +20,11 @@ Server::~Server() {
 		delete h_it->second;
 	}
 	SyscallWrap::closeWr(listen_sd_ DEBUG_INFO);
+	Server::serverSettingsMap::iterator s_it;
+	for (; s_it != settings_->end(); ++s_it) {
+		delete s_it->second;
+	}
```

Comment:
I forgot to delete these lines, since this settings were previously allocated dynamically but now it's done statically. Will update it